spec/mixlib/install/backend_spec.rb

```diff
@@ -187,10 +175,12 @@ def major_from(integration_version)
             }
           }
 
-          it_behaves_like "the right artifact info"
+          # it_behaves_like "the right artifact info"
```

Comment:
I would remove the tests instead of commenting out. The time we get to implement that we might want to do tests differently anyways :smile: